skins/oasis/css/core/ads.scss

```diff
@@ -232,8 +232,6 @@ body >a >img[src$="noad.gif"] {
 #WikiaAdInContentPlaceHolder {
 	position: absolute;
 	#INCONTENT_BOXAD_1 {
-		height: 250px;
-		width: 300px;
```

Comment:
And this is a tricky part... If I'm correct those sizes here were needed for good calculations. Now, before we get the ad from DFP we don't know if the height will be `250` or `600`. Without this sizes the calculations are always wrong because are made for height which equals `0`. It means `floatingMedrec` will calculate and decide the ad fits if it's 300x250 it'll probably fit but the higher one may not...

Let's think about how to solve it and we can talk after daily tomorrow. Unless you know how - then implement it please! :smiley_cat: